test/unit/testApplicantName.js

```diff
@@ -11,4 +15,107 @@ describe('name/index.js', () => {
             done();
         });
     });
+
+    describe('handlePost()', () => {
+        let ctx;
+        let errors;
+        let formdata;
+        const session = {};
+
+        beforeEach(() => {
+            session.form = {};
+        });
+
+        it('should return the ctx with the allowable characters for first and last name', (done) => {
+            ctx = {
+                firstName: 'Jason\'s with á, é, í, ó, ú, ý, Á, É, Í, Ó, Ú, Ý; ð, Ð',
```

Comment:
How does this test pass if you're specifying a name containing commas and semicolons, which are characters not allowed by the regex pattern in the schema?

Reply:
test was calling wrong method. it should have called validate method